Lib/asyncio/proactor_events.py

```diff
@@ -638,6 +637,8 @@ def __init__(self, proactor):
         if threading.current_thread() is threading.main_thread():
             # wakeup fd can only be installed to a file descriptor from the main thread
             signal.set_wakeup_fd(self._csock.fileno())
+        if self._debug:
+            logger.debug('Using proactor: %s', proactor.__class__.__name__)
```

Comment:
is this reachable? Can you write a test case that shows that it logs when in debug mode?

I worry that to set debug with `asyncio.run(coro(), loop_factory=asyncio.ProactorEventLoop, debug=True)` it won't log

Reply:
I had verified that `self._debug` does exist in `BaseProactorEventLoop`, but with `asyncio.run(..., debug=True)` it is still false

Reply:
Are there other EventLoop classes that have a debug log in their constructors?

Reply:
https://github.com/python/cpython/blob/6c52ada5512ccd1a0891c00cc84c7d3170d3328c/Lib/asyncio/selector_events.py#L52-L68

seems selector also has a debug log